doc/libpmempool/pmempool_feature_query.3.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEMPOOL_FEATURE_QUERY, 3)
+collection: libpmempool
+header: PMDK
+date: pmempool API version 1.3
+...
+
+[comment]: <> (Copyright 2018, Intel Corporation)
+
+[comment]: <> (Redistribution and use in source and binary forms, with or without)
+[comment]: <> (modification, are permitted provided that the following conditions)
+[comment]: <> (are met:)
+[comment]: <> (    * Redistributions of source code must retain the above copyright)
+[comment]: <> (      notice, this list of conditions and the following disclaimer.)
+[comment]: <> (    * Redistributions in binary form must reproduce the above copyright)
+[comment]: <> (      notice, this list of conditions and the following disclaimer in)
+[comment]: <> (      the documentation and/or other materials provided with the)
+[comment]: <> (      distribution.)
+[comment]: <> (    * Neither the name of the copyright holder nor the names of its)
+[comment]: <> (      contributors may be used to endorse or promote products derived)
+[comment]: <> (      from this software without specific prior written permission.)
+
+[comment]: <> (THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS)
+[comment]: <> ("AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT)
+[comment]: <> (LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR)
+[comment]: <> (A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT)
+[comment]: <> (OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,)
+[comment]: <> (SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT)
+[comment]: <> (LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,)
+[comment]: <> (DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY)
+[comment]: <> (THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT)
+[comment]: <> ((INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE)
+[comment]: <> (OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.)
+
+[comment]: <> (pmempool_feature_query.3 -- man page for toggle and query pool
```

Comment:
please add this and other function to `doc/Makefile`
and aliases to `doc/generated`